connectors/agentic-ai/src/main/java/io/camunda/connector/agenticai/a2a/client/model/result/A2AClientSendMessageResult.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. Licensed under a proprietary license.
+ * See the License.txt file for more information. You may not use this file
+ * except in compliance with the proprietary license.
+ */
+package io.camunda.connector.agenticai.a2a.client.model.result;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+import io.camunda.connector.agenticai.model.AgenticAiRecord;
+import io.camunda.connector.agenticai.model.message.content.Content;
+import java.util.List;
+
+@AgenticAiRecord
+public record A2AClientSendMessageResult(String responseId, List<Content> contents, TaskState state)
```

Comment:
Fine to do this in a follow-up PR, but should we already model this to return either `Task` or `Message` and align the properties with the spec? Something like

`Task`
- `id`
- `contextId`
- `status` (including state & message)
- `history` (list of messages - to be defined how many we'd include here)
- `artifacts`
- `metadata`

`Message`
- `role`
- `messageId`
- `taskId`
- `contextId`
- `referenceTaskIds`
- `parts` (mapped to our `Content` items)
- `metadata`
- `extensions` (not sure if needed already, but we could add it and configure it as `JsonInclude.Include.NON_EMPTY` to omit it from serialization)

Reply:
I wanted to do it later when we implement multi-turn conversations but I agree it is clearer to already return `Task` or `Message`. However, to keep it simple, let's only return the strict minimum properties for the moment. For example, we can include `history` in a dedicated PR, or we can leave out `extensions` for later.

Reply:
Sounds good! Happy with any variant as we'll definitely iterate on the structure, so we can also keep it like it is in this PR and address it in follow-ups if you prefer.